pkg/controller/controller_test.go

```diff
@@ -1390,48 +1391,6 @@ func TestProvision(t *testing.T) {
 	}
 }
 
-func TestProvisionWithMigration(t *testing.T) {
-	inTreePluginName := "kubernetes.io/gce-pd"
-	migrationDriverName := "pd.csi.storage.gke.io"
-	var requestBytes int64 = 100000
-
-	deletePolicy := v1.PersistentVolumeReclaimDelete
-	testcases := map[string]provisioningTestcase{
-		"should ignore in-tree with migration": {
```

Comment:
this test has been moved to the new `TestProvisionWithMigration` function and greatly simplified